src/clusterfuzz/_internal/protos/uworker_msg.proto

```diff
@@ -39,7 +39,9 @@ message AnalyzeTaskInput {
 }
 
 message FuzzTaskInput {
-  // TODO(https://github.com/google/clusterfuzz/issues/3008): Fill this in.
+  optional string sample_testcase_upload_key = 1;
```

Comment:
I wonder if we should even kill this `sample_testcase_upload` feature. Not sure how useful this is.